app/assets/stylesheets/govuk-component/_govspeak-html-publication.scss

```diff
@@ -109,6 +109,76 @@
         margin-top: ($gutter * 2) + $gutter-two-thirds;
       }
     }
+
+    table.financial-data {
+
+      .numeric {
+        text-align: right;
+        // allow uppercase letters in font variables as they're coming from govuk_frontend_toolkit
+        // scss-lint:disable NameFormat
+        font-family: $NTA-Light-Tabular;
+      }
+
+      // make all elements inside thead look the same
+      // (td can sometimes be used to overcome complexity of multiple lines in thead)
+      // and make all of thead and tfoot stand out
+      thead,
+      tfoot {
+        background-color: $grey-3;
+      }
+
+      thead th,
+      thead td {
+        font-weight: bold;
+      }
+
+      // don't make `th`s bold unless they are a section heading or a (sub)total row
+      tbody th,
+      tfoot th {
+        font-weight: normal;
+      }
+
+      // needs padding due to occasional background colour
+      // is intentionally small and not balanced with other padding
+      // as tables can be quite wide
+      tr > :first-child {
+        padding-left: $gutter-one-third / 2;
+      }
+
+      // add spacing so that groupings are clearer
+      tr.section-heading > * {
```

Comment:
This is my way of saving a few characters and making writing this less error-prone. Instead of `tr.foo th, tr.foo td` it saves the extra line by compressing that to `tr.foo > *`. If this was the two lines I could just write `.foo th, .foo td` but the single line only works if `.foo` is a `tr`. That's why I wouldn't want to change that to `.foo > *` as that makes it usable on the wrong elements.
So, the only way I could rewrite this to fit the linting configuration is by writing this instead:
```
.section-heading th,
.section-heading td {
}
```
But I'd prefer to disable the linting option.
This is the same for the other 4 occurrences of `tr > *` below.